cmd/create/machinepool/nodepool.go

```diff
@@ -76,7 +76,7 @@ func addNodePool(cmd *cobra.Command, clusterKey string, cluster *cmv1.Cluster, r
 		}
 
 		// Filter the available list of versions for a hosted machine pool
-		filteredVersionList := versions.GetFilteredVersionList(versionList, minVersion, clusterVersion)
+		filteredVersionList := versions.GetFilteredVersionList(versionList, minVersion, clusterVersion, false)
```

Comment:
Not a fan of adding bools to functions, would rather setup a strategy (https://refactoring.guru/design-patterns/strategy) or have the condition here separating two functions

Reply:
updated to have 2 different functions, not really worth a strategy. It should be clearer in terms of usage